### ISSUE TITLE
 "Multimaster node is not online: current status InMinor"

### ISSUE THREAD

**Issue by meagon** [postgrespro/postgres_cluster#31]
I've got this problem while use psql query, i do not known what was the problem
```
ERROR:  Multimaster node is not online: current status InMinor
```

**Comment by knizhnik** [comment]
It means that multimaster node is not able to establish connection with other nodes.
Please check log file.

**Comment by meagon** [comment]
both node are  starts welll, will if node lost connection as some time,  it will  auto reconnect?

**Comment by knizhnik** [comment]
Yes. Actually it is better to have at least 3 nodes, because in case of failure of any of two nodes there will be no quorum and remaining node can not proceed requests.

**Comment by meagon** [comment]
I  did it with 2 nodes,  and my wal sender process becomes in 
```
wal sender process postgres 192.168.56.103(37494) idle in transaction
```
when i use ```ps -ef  |grep postgres ```  show the command line  process name ?
is it an problem?
